crates/sui-core/src/authority_active.rs

```diff
@@ -47,7 +51,9 @@ use gossip::{gossip_process, node_sync_process};
 
 pub mod checkpoint_driver;
 use crate::authority_active::checkpoint_driver::CheckpointMetrics;
-use checkpoint_driver::checkpoint_process;
+use checkpoint_driver::{
+    checkpoint_process, get_latest_proposal_and_checkpoint_from_all, sync_to_checkpoint,
```

Comment:
Now that `get_latest_proposal_and_checkpoint_from_all, sync_to_checkpoin` are no longer only used in checkpoint driver, might want to move them to out of checkpoint driver.

Reply:
👍 - will add a TODO